src/modules/sessionManager.js

```diff
@@ -30,7 +29,7 @@ class SessionManager {
                             if (text.indexOf('SAMLRequest') !== -1) {
                                 // This is an html form page redirecting you to the default Identity Provider.
                                 // Let's orchestrate that request ourself
-                                
```

Comment:
Have you considered adding an `.editorconfig` file to the repo to prevent people from adding whitespace like this?

Reply:
I have no experience with that, hoping for someone else to open a PR :)